src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,121 @@
 package core.basesyntax;
 
+import java.util.Arrays;
+import java.util.NoSuchElementException;
+import java.util.Objects;
```

Comment:
dont use Arrays, Objects for learning perspective
```suggestion
import java.util.NoSuchElementException;
```